PoC/WiP: create separate talks/tutorials/posters pages (#605)
EuroPython/website: src/content/pages/programme.mdx

---
title: Programme
subtitle:

>     Learn more about all the activites at EuroPython 2024

Reviewer: ```suggestion
    Check out all the activities at EuroPython 2024
```

---

# Programme

> Welcome to the 23rd EuroPython. We’re the oldest and longest running volunteer-led Python programming conference on the planet! Join us in July in the vibrant Bohemian city Prague. We’ll be together, celebrating our shared passion for Python and its community!

Reviewer: ```suggestion
Welcome to the 23rd EuroPython. We’re the oldest and longest-running volunteer-led Python programming conference on the planet! Join us in July in the vibrant Bohemian city of Prague. We’ll be together, celebrating our shared passion for Python and its community!
```

A week of all things Python:

* Monday & Tuesday, 8 & 9 July: Tutorials & Workshops

> * Wednesday–Friday, 10-12 July: Conference talks & sponsor exhibition

Reviewer: ```suggestion
* Wednesday–Friday, 10–12 July: Conference talks & sponsor exhibition
```

* Saturday & Sunday, 13 & 14 July: Sprints

## Monday & Tuesday, 8 & 9 July

First two days are dedicated to the Tutorials and Workshops.

> We start at 09:00am at the [PCC](/venue). On the first two days everything happens on the first floor only.

Reviewer: ```suggestion
We start at 9:00 a.m. at the [PCC](/venue). On the first two days, all activities will take place on the first floor only.
```

> We have [16 tutorials](/tutorials), two special workshops: [Django Girls](https://djangogirls.org/en/prague/) on Monday and [Humble Data](https://humbledata.org/) on Tuesday.

Reviewer: ```suggestion
We are hosting [16 tutorials](/tutorials), and two special beginner workshops: [Django Girls](https://djangogirls.org/en/prague/) on Monday and [Humble Data](https://humbledata.org/) on Tuesday.
```

Two Summits – [C-API Summit](/c-api-summit) on Monday, and [WASM Summit](/wasm-summit) on Tuesday.

> **Keep in mind that in order to attend the Tutorials you need a [Tutorial or Combined](/tickets/#ticket-types) ticket. Summits and workshops have a separate registration.**

Reviewer: ```suggestion
**Keep in mind that in order to attend the Tutorials you need a [Tutorial or Combined](/tickets/#ticket-types) ticket. Summits and workshops require a separate registration.**
```

> Spaces are limited so act fast :)

Reviewer: ```suggestion
Spaces are limited, so act fast. :)
```

> If you are one of the speakers, we are also going to host a [Speaker's Dinner](/speakers-dinner) on Tuesday.

Reviewer: ```suggestion
If you are a speaker, we are also hosting a [Speaker's Dinner](/speakers-dinner) on Tuesday.
```

## Wednesday–Friday, 10-12 July

> Right after the opening session on Wednesday, we start with first of the [6 Keynotes](/keynotes).

Reviewer: ```suggestion
After the opening session on Wednesday, we will start with the first of the [6 Keynote talks](/keynotes).
```

> Then for the next three days, you can enjoy [110 Talks](/talks) running in 6 parallel tracks, and [9 Posters](/posters) happening during lunch breaks in the expo area.

Reviewer: ```suggestion
Then, for the next three days, you can enjoy [110 Talks](/talks) running in 6 parallel tracks, and [9 Posters](/posters) presented during lunch breaks in the expo area.
```

> All the main conference activities happen on those three days. On top of that you can visit our [Sponsor Exhibition](/sponsor), any of the other extra [Events](/events), including a [Social Event](/social-event) on Thursday.

Reviewer: ```suggestion
All the main conference activities occur on those three days. On top of that, you can visit our [Sponsor Exhibition](/sponsor) and any of the other extra [Events](/events), including a [Social Event](/social-event) on Thursday.
```

## Saturday & Sunday, 13 & 14 July

> For Saturday and Sunday we switch to a different venue, more suitable for [Sprints](/sprints).

Reviewer: ```suggestion
For Saturday and Sunday, we will switch to a different venue, more suitable for [Sprints](/sprints).
```

As per our tradition, the conference organisers will provide the rooms and

> facilities but the sprints are organised by YOU. It is a great chance to

Reviewer: ```suggestion
facilities, but the sprints are organised by YOU. It is a great chance to
```

contribute to open-source projects large and small, learn from each other, geek

> out and have fun.

Reviewer: ```suggestion
out, and have fun.
```
